### ISSUE TITLE
new problem with spaces and 'abbr.' element in items that were already proofread

### ISSUE THREAD

**Issue by upossekel** [srophe/e-gedsh#111]
I see that in entries that were proofread already, there are now new mistakes that seem to arise from the 'abbr.' element. The XML has been changed such that now at times the space occurs between two elements (one of which is the abbrev. element), so it is not longer displayed. I noticed this by chance and changed some items manually; but it seems this would better be addressed in a systematic way.

**Comment by wsalesky** [comment]
Yes, please give me links to specific examples, I suspect the problem is in the rendering, not the data. I changed how spacing was handled, sounds like this caused some issues.

**Comment by wsalesky** [comment]
@upossekel has this been resolved?

**Comment by upossekel** [comment]
There seem to be number if issues around the abbrev. element. I have manually fixed some of them (but inadvertently introduced a typo as I did so – still need to sort that out) and can just go through the list. I am a bit concerned about running another script on this – unless we can be sure what exactly are the implications. But I trust your expertise, and if you think running a script is the way to go, let’s do it. Let me follow up soon,  after I work on this some more. Thanks!
